tests/example_app/example/page1/page1a/templates/page_1a.html

```diff
@@ -0,0 +1,17 @@
+{% extends 'layout.html' %}
+
+{% block body %}
+  <br/>
+  <a href="/page-1/a">page 1 A</a>
+  <a href="/page-1/b">page 1 B</a>
+
+  <h2>Page 1A</h2>
+  This is an example application
+
+  <p>{{ static_url_for('.static', filename='app.css') }}</p>
+  <p>{{ static_url_for('page_1.page_1a.static', filename='app.css') }}</p>
+
+  <link type="text/css" rel="stylesheet"
+        href="{{ static_url_for('.static', filename='app.css') }}">
+
```

Comment:
Can you remove this extra new line?

Reply:
[27947c4 improved spacing consistency in jinja files](https://github.com/nickjj/flask-static-digest/pull/33/commits/27947c47ef46de2dbfc486e75705926ae6729b41)